google/cloud/iam/samples/iam_samples.cc

```diff
@@ -27,6 +27,35 @@
 #include <utility>
 
 namespace {
+
+void ExampleStatusOr(std::vector<std::string> const& argv) {
+  //! [example-status-or]
+  namespace iam = ::google::cloud::iam;
+  [](std::string const& project_id) {
+    iam::IAMClient client(iam::MakeIAMConnection());
+    int count = 0;
+    // The actual type of `service_account` is
+    // google::cloud::StatusOr<google::iam::admin::v1::ServiceAccount>, but
+    // we expect it'll most often be declared with auto like this.
+    for (auto const& service_account :
+         client.ListServiceAccounts(absl::StrCat("projects/", project_id))) {
```

Comment:
In examples, try to introduce as few libraries as possible, `"projects/" + project_id` should be familiar to all C++ programmers, `absl::StrCat` may not be.